<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Java 基础 on Shelby Lee</title>
    <link>https://shelbylee.github.io/topics/java-%E5%9F%BA%E7%A1%80/</link>
    <description>Recent content in Java 基础 on Shelby Lee</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <copyright>&amp;copy; 2018. All rights reserved.</copyright>
    <lastBuildDate>Thu, 19 Apr 2018 09:33:16 +0800</lastBuildDate>
    
	<atom:link href="https://shelbylee.github.io/topics/java-%E5%9F%BA%E7%A1%80/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>关于 try 和 finally 中的 return</title>
      <link>https://shelbylee.github.io/post/2018/04/19/</link>
      <pubDate>Thu, 19 Apr 2018 09:33:16 +0800</pubDate>
      
      <guid>https://shelbylee.github.io/post/2018/04/19/</guid>
      <description>关于 try 和 finally 中的 return 首先我们来看一段代码： public class Test { public static int inc() { int x = 1; try { return ++x; // 1* } catch (Exception e) { } finally { x++; } return x; } public static void main(String[] args) { System.out.println(inc()); } } 它的输出结果是多少呢</description>
    </item>
    
  </channel>
</rss>